packages/asset-swapper/src/utils/market_operation_utils/orders.ts

```diff
@@ -358,3 +359,32 @@ function createNativeOrder(fill: CollapsedFill): OptimizedMarketOrder {
         ...(fill as NativeCollapsedFill).nativeOrder,
     };
 }
+
+export function createSignedOrdersFromRfqtIndicativeQuotes(
+    quotes: RfqtIndicativeQuoteResponse[],
+): SignedOrderWithFillableAmounts[] {
+    return quotes.map(quote => {
+        return {
+            fillableMakerAssetAmount: quote.makerAssetAmount,
+            fillableTakerAssetAmount: quote.takerAssetAmount,
+            makerAssetAmount: quote.makerAssetAmount,
+            takerAssetAmount: quote.takerAssetAmount,
+            makerAssetData: quote.makerAssetData,
+            takerAssetData: quote.takerAssetData,
+            takerAddress: NULL_ADDRESS,
+            makerAddress: NULL_ADDRESS,
+            senderAddress: NULL_ADDRESS,
+            feeRecipientAddress: NULL_ADDRESS,
+            salt: ZERO_AMOUNT, // generatePseudoRandomSalt(),
```

Comment:
What's up with this comment? Can we leave as zero?

Reply:
Oh good eye.  This method was basically copied from https://github.com/0xProject/0x-monorepo/blob/110e1afa8e088288fb920a1f5689fe3a7f7b754a/packages/asset-swapper/src/utils/market_operation_utils/orders.ts#L338 , and I changed it to zero here, but preserved that call in the comments just in case it ended up not working for some reason.  At this point I can say it's working, but I'd love some feedback from @dorothy-zbornak on whether we should be concerned about this.

Reply:
I've cleaned up this function here: https://github.com/0xProject/0x-monorepo/pull/2555/commits/ad7868ebe10b902c04f34cd2cfdac64b72dfb8bc